### ISSUE TITLE
should --includestyles also include classes?

### ISSUE THREAD

**Issue by kohlhase** [brucemiller/LaTeXML#630]
What if I just want to read a class file ```foo.cls```? I have to make a ```foo.cls.ltxml``` with 
```
InputDefinitions('foo',type=>'cls',noltxml=>1); 
```
```foo.cls``` might something that just adapts ```article.cls``` minimally.

**Comment by brucemiller** [comment]
Hmm... it doesn't currently include classes, does it.  Probably wants to, although I think classes still may need some special treatment (see #622).  For example, some, but not all, cls files include a base article class; some just do it all from scratch.  But we'd want to include at least article or OmniBus class.

OTOH, your suggestion above is probably the only way to get full control, to be sure to LoadClass, if needed, and redefine a few semantic (likely frontmatter) macros.

**Comment by kohlhase** [comment]
Indeed, we need something like omnibus for the from-scratch case. But that does not seem to be a very common case. I would probably just issue a warning of the form 
```
loading a class ... proceed at your own risk ... you may want to make a wrapper that loads omnibus.cls.ltxml if the class does not load article.cls
```

**Comment by brucemiller** [comment]
Yep, makes sense. A catch is that we wouldn't know until *after* the .cls was read that it hadn't loaded any .cls.ltxml file.  I'm wondering if it's too late? OTOH, we don't want to assume article over report or book as base classes, if the .cls actually does read one of them!

**Comment by kohlhase** [comment]
You could just rig ```LoadClass``` to set a flag, and if this is unset at the end, you could generate the warning.

**Comment by kohlhase** [comment]
If that is before the fatals that will certainly come.
